extension-base/ctpGcfWrapper.js

```diff
@@ -0,0 +1,57 @@
+// Google Cloud Functions
+
+module.exports = {
+  ctpGcfWrapper: function(fn) {
+    return function(req, res) {
+      fn(req.body, ctpGcfResponse(res), console.log);
+    }
+  }
+}
+
+function ctpGcfResponse(res) {
+  return {
+    pass: passGcf(res),
+    errors: errorsGcf(res),
+    error: errorGcf(res),
+    updates: updatesGcf(res),
+    update: updateGcf(res),
+  }
+}
+
+function passGcf(res) {
+  return function() {
+    res.status(200).end();
+  }
+}
+
+function errorsGcf(res) {
+  return function(errors) {
+    res.status(400).json({
+      errors : errors
+    });
+  }
+}
+
+function errorGcf(res) {
```

Comment:
Unsure if we should/could combine these. After all it just seems to be about the "plural" of error(s) and action(s). I would just always go with plural as it doesn't feel too much asked for a consumer to "wrap" something in an Array I think as in `errors([error])` or `update([actions])`.

Reply:
Yes, it's just the plural.
At the API level I wanted to stay concise and only offer the plural. Here at SDK-ish level I feel like it's a nice helper. Especially in the `error` case, I guess 99% will never use the ability to return multiple errors.
But I'm open to other opinions :)